index.html

```diff
@@ -0,0 +1,31 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Awesome Books</title>
+</head>
+<body>
+    <div class="container">
+
+        <h1>Awesome Books</h1>
+        
+    <main>
+
+    </main>
+
+        <section class="add_book">
+            <!-- <form method="POST" action="_SELF" class="forum"> -->
+                <input type="text" placeholder="Title" id="title">
+                <input type="text" placeholder="Author" id="author">
+                <button type="submit" id="add">Add</button>
+            <!-- </form>  -->
```

Comment:
Great job team 🤝

- [ ] Kindly make sure that when the add button is clicked, it should not add automatically without any data being entered into the input fields. 
- [ ] I would suggest you create a form where you can add your author and title before you click the add button.